tools/cockpit.spec

```diff
@@ -374,10 +374,10 @@ The Cockpit component for managing storage.  This package uses Storaged.
 
 %package ostree
 Summary: Cockpit user interface for rpm-ostree
-%if 0%{?rhel}
-Requires: rpm-ostree-client >= 2015.11-1
-%else
+%if 0%{?fedora} > 0 && 0%{?fedora} < 24
```

Comment:
The point is that they want it both ways, one with rpm-ostree-client and one with rpm-ostree. If we don't want to do this they'll need to patch downstream.